lib/sproxyd.js

```diff
@@ -176,23 +175,35 @@ class SproxydClient {
         const newKey = key || keygen(this.cos, params);
         const req = this._createRequestHeader(method, headers, newKey, params,
                                               log);
+        const host = this.getCurrentBootstrap();
         if (stream) {
             headers['content-length'] = size;
-            const request = _createRequest(req, log, err => {
+            const request = _createRequest(req, log, (err, response) => {
                 if (err) {
-                    log.error('PUT chunk to sproxyd', { error: err });
+                    log.error('putting chunk to sproxyd', { host, key: newKey,
+                        error: err });
                     return callback(err);
                 }
                 // We return the key
-                log.debug('stored to sproxyd', { newKey });
+                log.debug('stored to sproxyd', { host, key: newKey,
+                    statusCode: response.statusCode });
                 return callback(null, newKey);
             });
             stream.pipe(request);
             request.on('end', () => request.end);
             log.debug('finished sending PUT chunks to sproxyd');
         } else {
             headers['content-length'] = 0;
-            const request = _createRequest(req, log, callback);
+            const request = _createRequest(req, log, (err, response) => {
+                if (err) {
+                    log.error('error in _handleRequest', { host,
```

Comment:
This log message could be `error sending sproxyd request` and a method property as _handleRequest would be appropriate.